packages/beacon-node/src/util/dataColumns.ts

```diff
@@ -243,7 +243,7 @@ export function getDataColumnSidecars(
   signedBlockHeader: SignedBeaconBlockHeader,
   kzgCommitments: deneb.KZGCommitment[],
   kzgCommitmentsInclusionProof: fulu.KzgCommitmentsInclusionProof,
-  cellsAndKzgProofs: [Uint8Array[], Uint8Array[]][]
+  cellsAndKzgProofs: {cells: Uint8Array[]; proofs: Uint8Array[]}[]
```

Comment:
I like this new data structure, it's so straightforward and make it easy for everybody to understand